mock/py/mockbuild/plugins/ccache.py

```diff
@@ -47,6 +48,9 @@ def __init__(self, plugins, conf, buildroot):
     @traceLog()
     def _ccacheBuildHook(self):
         self.buildroot.doChroot(["ccache", "-M", str(self.ccache_opts['max_cache_size'])], shell=False)
+        # zero ccache stats
+        getLog().info("Zero ccache stats:")
+        self.buildroot.doChroot(["ccache", "--zero-stats"], printOutput=True, shell=False)
```

Comment:
Could you please wrap this command into a separate method, and call it twice?

Reply:
Could we ignore the exit status?  Just in case the `--zero-stats` option was unavailable in the target chroot.

Reply:
I made the feature opt-in instead.